todocli/datetime_parser.py

```diff
@@ -16,24 +16,111 @@ def parse_day_month(input_str):
     return day, month
 
 
+def parse_day_month_american(input_str):
+    split_str = input_str.split("/")
+    month = int(split_str[0])
+    day = int(split_str[1])
+    return day, month
+
+
+def add_day_if_datetime_is_in_past(dt: datetime) -> datetime:
```

Comment:
I don't think the function name has to be this descriptive. A simple name like `add_duedate` or something like that would be easier. Adding another day should be in the function logic (which it is).

Reply:
I'd rename it to "add_day_if_past", would you agree with that?